images/Dockerfile.ubuntu-20-lts-arm-rpi

```diff
@@ -0,0 +1,63 @@
+ARG BASE_IMAGE=ubuntu:20.04
+
+FROM $BASE_IMAGE
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN echo "deb http://ports.ubuntu.com/ubuntu-ports/ focal main universe restricted multiverse" >> /etc/apt/sources.list.d/raspberrypi.list
+RUN apt-get update \
+    && apt-get install -y software-properties-common \
+    && add-apt-repository ppa:oibaf/test \
+    && apt-get update \
+    && apt-get install -y \
+    conntrack \
+    console-data \
+    coreutils \
+    curl \
+    debianutils \
+    dosfstools \
+    dracut \
+    dracut-network \
+    e2fsprogs \
+    gawk \
+    grub-efi-arm64-bin \
+    grub2-common \
+    haveged \
+    iproute2 \
+    iptables \
+    jq \
+    linux-firmware-raspi2 \
+    linux-raspi \
+    lldpd \
+    nano \
+    neovim \
+    open-vm-tools \
+    openssh-server \
+    parted \
+    policykit-1 \
+    rpi-eeprom \
+    rsync \
+    snapd \
+    snmpd \
+    squashfs-tools \
```

Comment:
dracut complained that its missing sgdisk, so we may need to install it as agetn will use it for partition stuff.